### DISCUSSION TITLE
Add a open recently button to the menu

### DISCUSSION THREAD

**Discussion by Gus-The-Forklift-Driver** [mbrlabs/Lorien#54] [Ideas]
Pretty self explanatory, add a button that stores 5-10 last opened files for quick access.
(when using save-as add the original file to the list)

**Comment by bennyschirm** [comment]
Good idea! I suggest an additional menu item which keeps track of the last ~10 opened files.

**Comment by ghost** [comment]
Yes,needs to not only remember recent files, but even store the last directory path from the last save.
Once I close Lorien and open it again, it doesn't even remember the last file it was working on, nor even the directory where the last save was.
Could also have a setting to open up the more recent file.
.
It's kind of the basics needed by all apps, the UI and usability need to be prioritized before features are added.

**Comment by mbrlabs** [comment]
Something like that has already been implemented on the main branch (not in the latest release yet). See the discussion in https://github.com/mbrlabs/Lorien/issues/67 and the [PR](https://github.com/mbrlabs/Lorien/pull/78).
